ubuntu.json

```diff
@@ -153,6 +154,13 @@
   ],
   "post-processors": [
     {
+      "type": "compress",
+      "only": ["vmware-iso"],
+      "output": "{{.BuilderType}}_{{user `version`}}.tar.gz",
+      "keep_input_artifact": false,
+      "compression_level": 1
+    },
+    {
```

Comment:
You are correct, I will go back and take out that section, this weekend
seems like a good time to learn how to stage hunks instead of entire files,
& I will also do a rebase instead of the merge. Thanks for the feedback and
I'll clean it up and update the request.
On Jan 8, 2016 1:14 AM, "Rickard von Essen" <notifications@github.com>
wrote:

> In ubuntu.json
> <https://github.com/boxcutter/ubuntu/pull/40#discussion_r49163365>:
>
> > @@ -153,6 +154,13 @@
> >    ],
> >    "post-processors": [
> >      {
> > +      "type": "compress",
> > +      "only": ["vmware-iso"],
> > +      "output": "{{.BuilderType}}_{{user `version`}}.tar.gz",
> > +      "keep_input_artifact": false,
> > +      "compression_level": 1
> > +    },
> > +    {
>
> This change is not in the description of the PR and I doubt that it is
> intended.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/boxcutter/ubuntu/pull/40/files#r49163365>.
>